src/features/inventory/index.js

```diff
@@ -51,10 +69,11 @@ class Inventory extends Component {
       <div className='flex-row inventory__container'>
         {
           !disabled &&
-            <Button
+            <InventoryButton
               small={sideMenu}
               indicator={newItemIndicator}
               onClick={this._toggleInventory.bind(this)}
+              onKeyDown={this.handleKeyDown.bind(this)}
```

Comment:
We should be able to avoid having to deal with `bind` altogether if we use arrow functions.

Reply:
I agree with you. In fact, I usually go with something like this:

```
import React, { Component } from 'react';

class Container extends Component {
  handleClick = (event) => {
    // do your thing!
  }

  render() {
    return (
      <button onClick={this.handleClick}>Click me!</button>
    );
  }
}
```

No `.bind()` functions anywhere. Nothing.

But when I did, eslint complains that I'm not following standard:
```
Compiled with warnings.

./src/features/inventory/index.js
  Line 52:  Unexpected 'this'  no-invalid-this
  Line 53:  Unexpected 'this'  no-invalid-this
```

Culprit: https://github.com/ASteinheiser/react-rpg.com/blob/master/.eslintrc.yml#L13

Uh... I'm not well-versed in ESLint, but I found solutions here:
1. remove `no-invalid-this: warn` at the risk of some loss in coding standard and error checks
2. use [`eslint-plugin-babel`](https://github.com/babel/eslint-plugin-babel), then configure the ESLint to use `babel/no-invalid-this` ([as seen here](https://medium.com/@chestozo/fixing-eslint-no-invalid-this-error-for-fat-arrow-class-methods-a56908ca8bb6)).
3. use [`react/jsx-no-bind`](https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-no-bind.md), but then you'll end up having to declare all arrow function props outside of the JSX. [I have a WIP branch on this](https://github.com/ASteinheiser/react-rpg.com/compare/master...KajiTetsushi:feature/no-invalid-this-eslint-babel). I still have a long way to go!

i.e. 
you shouldn't do this
```
<button onClick={() => setWhateverState(true)} />
```

you should do this
```
const handleClick = useCallback(
  () => setWhateverState(true),
  []
);

<button onClick={handleClick} />
```

[There is a valid reason for it in fighting unnecessary component re-renders](https://stackoverflow.com/a/53246229/1933636) ([more discussion](https://stackoverflow.com/a/54372172/1933636)). In my experiences, I make it a point to not make this happen in production code, too. I'd wager this is why I experience some input lag when I run the game locally.

...
...
I am very sorry for the long story, but please bear with me. I'm trying to get my point across.